res/schema.xml

```diff
@@ -582,4 +582,28 @@ reapplying those migrations.
       UPDATE library SET filetype='aiff' WHERE filetype='aif';
     </sql>
   </revision>
+  <revision version="40" min_compatible="3">
+    <description>
+      Suggestion tables added.
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS suggestion_library (
+        id INTEGER,
+        artist varchar(48),
+        title varchar(48),
+        playcount INTEGER,
+        match FLOAT,
+        duration FLOAT,
+        location varchar(512));
+    </sql>
+  </revision>
+    <revision version="41" min_compatible="3">
+    <description>
+      Playcount and match column added to library table for sorting.
+    </description>
+    <sql>
+      ALTER TABLE library ADD COLUMN playcount float DEFAULT NULL;
+      ALTER TABLE library ADD COLUMN match float DEFAULT NULL;
```

Comment:
I don't see any reason for adding an unrelated, generic `match` column to the `library` table. It doesn't belong there. If you want to sort than use a join or a view.

Reply:
This is the match value returned by last.fm. It should also be renamed to make this clear.

Reply:
It still doesn't belong into this table.

Reply:
Right, it is the wrong table.

Reply:
These two columns are deleted now.